library/core/class.format.php

```diff
@@ -1436,6 +1436,12 @@ private static function embedReplacement($url) {
             'Twitch' => [
                 'regex' => ['/https?:\/\/(?:www\.)?twitch\.tv\/([\w]+)/i']
             ],
+            'TwitchRecorded' => [
+                'regex' => ['/https?:\/\/(?:www\.)?twitch\.tv\/videos\/(\w+)$/i']
+            ],
+            'Twitch' => [
+                'regex' => ['/https?:\/\/(?:www\.)?twitch\.tv\/([\w]+)$/i']
```

Comment:
Why do we need to regex entries for Twitch?
line 1436 & 1443

Reply:
We don't! :P